lib/simplifyExpression/division/index.js

```diff
@@ -1,18 +1,16 @@
 'use strict';
 
-const MathChangeTypes = require('./MathChangeTypes');
-const NodeCreator = require('./util/NodeCreator');
-const NodeStatus = require('./util/NodeStatus');
-const NodeType = require('./util/NodeType');
-const TreeSearch = require('./util/TreeSearch');
+const ChangeTypes = require('../../ChangeTypes');
+const Node = require('../../Node');
+const TreeSearch = require('../../TreeSearch');
 
 // Searches for and simplifies any chains of division or nested division.
-// Returns a NodeStatus object
-const simplifyDivisionSearch = TreeSearch.preOrder(simplifyDivison);
+// Returns a Node.Status object
+const search = TreeSearch.preOrder(division);
 
-function simplifyDivison(node) {
-  if (!NodeType.isOperator(node) || node.op !== '/') {
-    return NodeStatus.noChange(node);
+function division(node) {
```

Comment:
another thing I feel weird about is that the function names are less descriptive - to understand what this function does, you can't just look at the function name anymore you have to know the folder hierarchy (or read the docstring I guess)